control-plane/agents/src/bin/core/controller/io_engine/v1/replica.rs

```diff
@@ -113,6 +113,29 @@ impl crate::controller::io_engine::ReplicaApi for super::RpcClient {
         Ok(())
     }
 
+    #[tracing::instrument(name = "rpc::v1::replica::resize", level = "debug", skip(self), err)]
+    async fn resize_replica(&self, request: &ResizeReplica) -> Result<Replica, SvcError> {
+        let response = self
+            .replica()
+            .resize_replica(request.to_rpc())
+            .await
+            .context(GrpcRequestError {
+                resource: ResourceKind::Replica,
+                request: "resize_replica",
+            })?;
+
+        if let Some(rpc_replica) = response.into_inner().replica {
```

Comment:
Why is this optional? Perhaps not up to date with protobuf?

Reply:
> Why is this optional? Perhaps not up to date with protobuf?

Yeah, updated now based on protobuf changes.